src/components/filemanager-card.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import {
+  Card,
+  CardHeader,
+  CardMeta,
+  Progress,
+  Image,
+  Icon
+} from 'semantic-ui-react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+class Filemanagercard extends Component {
+  constructor (props) {
+    super(props)
+  }
+  render () {
+    const { filemanagername, contentSize, maxSpace } = this.props
+    let percentage = (contentSize * 100) / maxSpace
+    return (
+      <Card as={Link} to={`/file-manager/${filemanagername}`}>
+        <Card.Content>
+          <Image
+            floated='right'
+            size='mini'
+            src='https://img2.pngio.com/yellow-folder-transparent-png-clipart-free-download-ywd-yellow-folder-png-2400_1729.png'
```

Comment:
why is the src of image hard coded?
it should be in the constants file. If it is not present then make a new file name constants

Reply:
it is a dummy image used just to check UI, We will be displaying unique images corresponding to the filemanager instance.
This image will not be used afterwords therefore i am not including it in constants for now.